### PR TITLE
Standardizing MRMS Options

### PR DESCRIPTION
**Pull request by Windchild292**
This standardizes MRMS into immediately using MassRepairConfiguredOptions. This is required for a few future MRMS stories.

### CODE REVIEW

File: MekHQ/src/mekhq/service/MassRepairConfiguredOptions.java [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+    }
+
+    public void setHasActiveMassRepairOption(boolean hasActiveMassRepairOption) {
+        this.hasActiveMassRepairOption = hasActiveMassRepairOption;
+    }
+    //endregion Getters/Setters
+
+    public List<MassRepairOption> getActiveMassRepairOptions() {
+        if (!hasActiveMassRepairOption()) {
+            return new ArrayList<>();
```
Comment (sixlettervariables) [inline MekHQ/src/mekhq/service/MassRepairConfiguredOptions.java:186]: If this should be immutable then prefer `Collections.emptyList()`. (realizing this was perhaps inherited code)

File: MekHQ/src/mekhq/service/MassRepairOption.java [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+                mro.setSkillMin(Integer.parseInt(mroItemNode.getTextContent().trim()));
+            } else if (mroItemNode.getNodeName().equalsIgnoreCase("skillMax")) {
+                mro.setSkillMax(Integer.parseInt(mroItemNode.getTextContent().trim()));
+            } else if (mroItemNode.getNodeName().equalsIgnoreCase("btnMin")) {
+                mro.setBthMin(Integer.parseInt(mroItemNode.getTextContent().trim()));
+            } else if (mroItemNode.getNodeName().equalsIgnoreCase("btnMax")) {
+                mro.setBthMax(Integer.parseInt(mroItemNode.getTextContent().trim()));
+            }
+
+            MekHQ.getLogger().info(String.format("massRepairOption %d.%s\n\t%s",
```
Comment (sixlettervariables) [inline MekHQ/src/mekhq/service/MassRepairOption.java:175]: Should this instead be `debug`?

**Review by sixlettervariables — APPROVED** [review]
Two quick comments, but nothing holding this up.

File: MekHQ/src/mekhq/service/MassRepairOption.java (hunk 2 above)
Comment (Windchild292) [reply MekHQ/src/mekhq/service/MassRepairOption.java:175]: We currently use info for this and campaign options... but I'd say yes

**Pull request merged**